src/main/java/com/salesforce/mirus/KafkaMonitor.java

```diff
@@ -89,6 +96,51 @@
         config.getEnablePartitionMatching()
             ? SourcePartitionValidator.MatchingStrategy.PARTITION
             : SourcePartitionValidator.MatchingStrategy.TOPIC;
+    this.routers = this.validateTransformations(config.transformations());
+    this.topicNamePrefix = config.getDestinationTopicNamePrefix();
+    this.topicNameSuffix = config.getDestinationTopicNameSuffix();
+  }
+
+  private List<Transformation<SourceRecord>> validateTransformations(
+      List<Transformation<SourceRecord>> transformations) {
+    List<Transformation<SourceRecord>> regexRouters = new ArrayList<>();
+    for (Transformation<SourceRecord> transform : transformations) {
+      String transformName = transform.getClass().getName();
+      if (transform instanceof RegexRouter) {
+        regexRouters.add(transform);
+        // Slightly awkward check to see if any other routing transforms are configured
+      } else if (transform.getClass().getName().contains("Router")) {
```

Comment:
Do you mean to be checking the full name of the class including package, or are you more interested in the class' simple name? In any case, you already obtained the name once already as `transformName`.